nautobot/extras/models/groups.py

```diff
@@ -64,17 +64,13 @@ class DynamicGroup(OrganizationalModel):
 
     clone_fields = ["content_type", "filter"]
 
-    # This is used as a `startswith` check on field names, so these can be
-    # explicit fields or just substrings.
+    # This is used as a `startswith` check on field names, so these can be explicit fields or just
+    # substrings.
     #
-    # Currently this means skipping "search", custom fields, and custom relationships.
-    #
-    # FIXME(jathan): As one example, `DeviceFilterSet.q` filter searches in `comments`. The issue
-    # really being that this field renders as a textarea and it's not cute in the UI. Might be able
-    # to dynamically change the widget if we decide we do want to support this field.
```

Comment:
Isn't this comment still relevant regarding `comments`?

Reply:
Yes. The widget is weird. Comments are weird. They are exact match only and are better covered by search.

Reply:
I think @glennmatthews' comment is shouldn't we leave the FIXME in here on the _why_ we're skipping computed fields and comments.